packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -14,48 +14,89 @@ import type { NodeCheckboxRenderProps } from "@itwin/core-react";
 import type { IPresentationTreeDataProvider } from "@itwin/presentation-components";
 import type { VisibilityTreeFilterInfo } from "./Common";
 
+/**
+ * Props for visibility tree node renderer.
+ * @public
+ */
+export interface VisibilityTreeRendererProps {
+  /**
+   * Specifies whether the icon at the left of the node label should be rendered.
+   */
+  iconsEnabled: boolean;
+  /**
+   * Specifies whether node description should be enabled.
+   */
+  descriptionEnabled: boolean;
+  /**
+   * Defines the offset in pixels of how much each hierarchy level should be offset to the right from the checkbox.
+   * Defaults to `20`.
+   */
+  levelOffset?: number;
+  /**
+   * Defines the size in pixels of the expansion toggle. It is used to keep same hierarchy nodes with children and nodes without children in the same line.
+   * @note This value applies only to the leaf nodes.
+   * Defaults to `24`.
+   */
+  expansionToggleWidth?: number;
```

Comment:
From the docs it doesn't look like consumers would need to change this value. Probably not needed in public API?

Reply:
removed